cpp/console/console.h

```diff
@@ -34,6 +35,8 @@ class Console : InputHandler, TickHandler, HudHandler, ResizeHandler {
 	Buf buf;
 	Font font;
 
+	keybinds::KeybindContext keybindContext;
```

Comment:
sorry that i noticed this that late, please name the variable `keybind_context`, but the class name is perfect :wink: